functions/public/Invoke-WPFGetIso.ps1

```diff
@@ -138,7 +173,7 @@ function Invoke-WPFGetIso {
             $imageName = $_.ImageName
             $sync.MicrowinWindowsFlavors.Items.Add("$imageIdx : $imageName")
         }
-        $sync.MicrowinWindowsFlavors.SelectedIndex = 0
+        $sync.MicrowinWindowsFlavors.SelectedIndex = 5
```

Comment:
What if the Windows image selected doesn't contain 11 indexes? What if the Pro edition isn't the sixth index?

This can fail by throwing an exception. We shouldn't do it this way.